src/routes/safe/components/Apps/AddAppForm.tsx

```diff
@@ -102,7 +102,7 @@ const curriedSafeAppValidator = memoize((appList) => async (value: string) => {
   }
 })
 
-const composeValidatorsApps = (...validators: Function[]) => (value, values, meta) => {
+const composeValidatorsApps = (...validators) => (value, values, meta) => {
```

Comment:
why the type was removed?

Reply:
Got the following error:
```
ESLint: 
Don't use `Function` as a type. The `Function` type accepts any function-like value.
It provides no type safety when calling the function, which can be a common source of bugs.
It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
If you are expecting the function to accept certain arguments, you should explicitly define the function shape.
(@typescript-eslint/ban-types)
```
by removing it, did the trick.

This error was preventing me from committing changes.